src/navigation/navigationstate.cpp

```diff
@@ -33,6 +33,22 @@ namespace {
 
     constexpr double Epsilon = 1E-7;
 
+    // A NavigationState is an object describing an exact camera position and rotation,
+    // in a certain reference frame (per default, the one of the specified Anchor node).
+    // It can be used to set the same camera position at a later point in time, or
+    // navigating to a specific camera position usign the pathnavigation system.
+    //
+    // The camera rotation is specified using Euler angles, in radians. It is also
+    // possible to specify a node to be used as Aim, but note that this will not affect
+    // the actual camera position or view direction.
+    //
+    // To get the current navigation state of the camera, use the
+    // `openspace.navigation.getNavigationState()` function in the Scripting API.
```

Comment:
Yeah, I thought a bit about that too. However, it felt very relevant for this specific documentation 🤔 

Also, if we change the syntax of the function, this documentation is still within the code base.  So if you search for where the function is being used, it should appear. And hence I don't personally think it's a problem keeping it up to date. What do you think?

Reply:
In this case, we _should_ be able to use the `#` to link to the documentation of the other function. I guess it would be `#openspace::scripting::getNavigationState`, but I'm not sure about the exact name

Reply:
I think that if it is possible to refer somewhere and not have to specify it manually we should strive to do that.